### DISCUSSION TITLE
Can't open sample project?

### DISCUSSION THREAD

**Discussion by muchogrande** [pafuhana1213/KawaiiPhysics#121] [Q&A] [answered]
Hi, trying to open the sample project and am told :

![Screenshot 2024-06-20 113544](https://github.com/pafuhana1213/KawaiiPhysics/assets/13224511/c8ec3ef8-ebbc-448d-8ad3-f61ee49b9fd4)

Rebuild (clicking yes) fails. Not sure what to do?

**Comment by pafuhana1213** [comment, marked as answer]
Hi, @muchogrande 

If you can't build c++ project, you should use the following plugin binary.
https://github.com/pafuhana1213/KawaiiPhysics/releases

thanks!
